Database/CleanArchive.py

```diff
@@ -14,20 +17,22 @@ def clean(all=True):
     '''
     updates = Counter()  # Track metrics for user
     failures = Counter()
+    duplicates = Counter()
     query = {}
     if not all:
         query = {
-            'transcript_str_length': {'$exists': False},
+            # 'transcript_str_length': {'$exists': False},
```

Comment:
It's not the cleanest query, but this makes sure we don't re-clean episodes we have already cleaned, speeding up execution.